frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -162,3 +162,7 @@ export const removeTopologyResourceConnection = (edge: Edge): Promise<any> => {
 
   return removeResourceConnection(source, target);
 };
+
+export const isOperatorBackedNode = (element: Node): boolean =>
+  element.getData().resources?.['isOperatorBackedService'] ||
+  element.getParent().getData().resources?.['isOperatorBackedService'];
```

Comment:
`getData()` could return null. We should likely check any upward parent not just the immediate parent.